src/Tribe/Main.php

```diff
@@ -1594,9 +1594,7 @@ public function saved_organizers_dropdown( $current = null, $name = 'organizer[O
 		 */
 		public function admin_body_class( $classes ) {
 			global $current_screen;
-			if ( isset( $current_screen->post_type ) &&
-				 ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' )
-			) {
+			if ( isset( $current_screen->post_type ) && ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' ) ) {
```

Comment:
So @borkweb the thing is that my Code Sniffer was complaning about multiple lines in the conditionals.

When I change something I will work with Yoda Conditions then, just didn't occur to me that we would do it as of right now.